orchestrator/gorc/src/commands/keys/eth.rs

```diff
@@ -6,28 +6,28 @@ mod recover;
 mod rename;
 mod show;
 
-use abscissa_core::{Command, Options, Runnable};
+use abscissa_core::{Command, Clap, Runnable};
 
-#[derive(Command, Debug, Options, Runnable)]
+#[derive(Command, Debug, Clap, Runnable)]
 pub enum EthKeysCmd {
-    #[options(help = "add [name]")]
+    #[clap(name = "add")]
```

Comment:
I believe `clap` will auto-infer these names based on the variant names, i.e. `Add` => `add`, so this (and the ones below) can be removed.